src/Entity/Thing.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace App\Entity;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Id\UuidGenerator;
+use Doctrine\ORM\Mapping as ORM;
+use Ramsey\Uuid\Generator\CombGenerator;
+use Ramsey\Uuid\Uuid;
+
+/**
+ * @ORM\Entity(repositoryClass="App\Repository\ThingRepository")
+ */
+class Thing
+{
+    /**
+     * @ORM\Id()
+     * @ORM\Column(type="guid")
+     */
+    private string $id;
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private string $name;
+
+    /**
+     * @ORM\OneToMany(targetEntity="App\Entity\Order", mappedBy="thing")
+     */
+    private $orders;
+
+    public function __construct(
+        string $name
+    ) {
+        $this->name = $name;
+        $this->id = Uuid::uuid4()->toString();
+        $this->orders = new ArrayCollection();
+    }
+
+    public function getId(): ?string
+    {
+        return $this->id;
+    }
+
+    public function getName(): ?string
+    {
+        return $this->name;
+    }
+
+    public function setName(string $name): self
+    {
+        $this->name = $name;
+
+        return $this;
+    }
+
+    /**
+     * @return Collection|Order[]
+     */
+    public function getOrders(): Collection
+    {
+        return $this->orders;
```

Comment:
Ich würde die Collection nicht nach außen geben und lieber einfach nur ein Array zurückgeben.
`return $this->orders;`